lib/floe/cli.rb

```diff
@@ -18,19 +19,25 @@ def run(args = ARGV)
           opts[:credentials] == "-" ? $stdin.read : opts[:credentials]
         elsif opts[:credentials_file_given]
           File.read(opts[:credentials_file])
+        else
+          {}
         end
 
       workflows =
         workflows_inputs.each_slice(2).map do |workflow, input|
-          context = Floe::Workflow::Context.new(opts[:context], :input => input, :credentials => credentials)
-          Floe::Workflow.load(workflow, context)
+          create_workflow(workflow, opts[:context], input, credentials)
         end
 
-      Floe::Workflow.wait(workflows, &:run_nonblock)
+      output_streams = create_loggers(workflows, opts[:segment_output])
+
+      puts "Checking #{workflows.count} workflows..."
```

Comment:
This won't be in the "real" application logs now though, right (since this is moved to cli, which is a dev tool)?  That was my concern.  Maybe it's not necessary in the application logs?

Reply:
this is the cli. so yes, it will only go to stdout.
All entries in this file go to stdout via `puts`

Reply:
I guess what I'm asking is before this change, this particular log line went to the application logs and now it doesn't.  I'm not sure how useful that is - maybe @agrare has opinions here.

Reply:
Could we set the Floe.logger to stdout (if it isn't already) and
1. Keep the .wait logger the way it was
2. Use that instead of puts in cli.rb

Reply:
Most logging we have is feedback for individual workflows (mostly step start/complete)

The cli uses `State.wait` to orchestrate which workflows are run. I think of this as part of the command line interface. So going away from stdout seems strange to me. (asid: I do want to refactor this method to go even based, but that is a future action)

So is the thought is to convert all cli `puts` statements to log statements?